apps/portal/src/app/engine/v3/faq/page.mdx

```diff
@@ -3,11 +3,11 @@ import { Callout, Details } from "@doc";
 # Engine FAQs
 
 <Details summary="How is pricing calculated for Engine Cloud?">
-Pricing is calculated through the number of write requests (ex: /v1/write/contract) made through the Engine API. Requests cost $1 per 1,000 requests. Read requests made through Engine API are free, within the RPC limits/plan. 
+Pricing is calculated through the number of write requests (ex: /v1/write/contract) and sign requests (ex: /v1/sign/transaction) made through the Engine API. Write and sign requests cost $1 per 1,000 requests. Read requests made through Engine API are free, within the RPC limits/plan. 
 
 For transactions through server wallets paid through the user's thirdweb account, users will pay a 5% premium on the gas fee of each transaction completed on any mainnet as part of account abstraction fees. There are no gas charges for testnets.
 
-**Please note: Legacy pricing plans may be subject to the 10% fee instead of 5%. Please update your pricing plan to avoid being subject to the 10% fee.** 
+**Please note: Legacy pricing plans may be subject to the 10% fee specified when signed up instead of 5%. If you are on a Legacy Starter plan, we recommend updating your pricing plan to be subject to the 5% fee and take advantage of other benefits. [See pricing page](https://thirdweb.com/pricing)** 
```

Comment:
The intent is good here but seems a bit wordy. Maybe just a footnote that legacy plans may be on older fee structure, and  to view the pricing page for new pricing and other benefits.